include/open62541/plugin/nodestore.h

```diff
@@ -645,6 +652,7 @@ typedef UA_StatusCode
 typedef struct {
     UA_NodeHead head;
     UA_Boolean executable;
+    UA_Boolean userExecutable;
```

Comment:
The UserExecutable attribute is computed dynamically for each session. Don't store it in the node.